### DISCUSSION TITLE
Single-DB multi-tenancy

### DISCUSSION THREAD

**Discussion by gnimmelf** [directus/directus#3409] [Feature Requests]
Hi folks,

Just a thought;

To mimic single-DB multi-tenancy, could it be an idea to have a preset on par with "Reset system permissions" on the role permissions to set all the system collections? - Like an "owner only"-policy for the given (custom) role.

E.g for the system collections `Directus files`, there would be `create::full access`, and custom permission 
```
{
    "uploaded_by": {
        "_eq": "$CURRENT_USER"
    }
}
```
for the rest.

Disclaimer! 2nd day looking at Directus, and still learning.

Cheers!
Flemming

**Comment by benhaynes** [comment]
I like it! Yeah, there are pretty much limitless possibilities for how to do MT in this new rule-based permission system. Really good to have references like this to guide people... thanks!

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
